src/test/scala/com/github/codelionx/distod/ResultFileParsing.scala

```diff
@@ -0,0 +1,130 @@
+package com.github.codelionx.distod
+
+import java.io.{BufferedReader, FileReader}
+
+import com.github.codelionx.util.TryWithResource._
+
+import scala.collection.immutable.Set
+import scala.jdk.StreamConverters._
+
+
+object ResultFileParsing {
+
+  object ODType {
+    sealed trait Type
+    case object Constant extends Type
+    case object Equivalency extends Type
+  }
+
+  sealed trait ODResult
+  final case class ConstantODResult(context: Set[String], constantAttribute: String) extends ODResult
+  final case class EquivalencyODResult(
+      context: Set[String],
+      attribute1: String,
+      attribute2: String,
+      reverse: Boolean = false
+  ) extends ODResult
+
+  def readAndParseDistodResults(path: String): Seq[ODResult] =
+    withResource(new BufferedReader(new FileReader(path))) { reader =>
+      val lines = reader.lines().toScala(LazyList)
+      DistodResultParser.parseAll(lines)
+    }
+
+  def readAndParseFastodResults(path: String): Seq[ODResult] =
+    withResource(new BufferedReader(new FileReader(path))) { reader =>
+      val lines = reader.lines().toScala(LazyList)
+      FastodResultParser.parseAll(lines)
+    }
+
+  trait ResultParser {
+
+    protected val separator = ":"
+    protected val compatibilitySymbol = "~"
+    protected val constantSymbol = "[] ↦"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Non ascii characters are not allowed](https://app.codacy.com/app/CodeLionX/distod/pullRequest?prid=4634600)